src/homelogstore/log_store_family.cpp

```diff
@@ -319,14 +325,18 @@ sisl::status_response LogStoreFamily::get_status(const sisl::status_request& req
     response.json["log_dev"] = m_log_dev.get_status(request).json;
 
     // All logstores in range [lower_logstore_id, higher_logstore_id] if range is provided
-    logstore_id_t lower_logstore_id = (request.json.contains("min_logstore_id")) ? static_cast<logstore_id_t > (std::stoul(request.json["min_logstore_id"].get<std::string>())) : 0;
-    logstore_id_t higher_logstore_id = (request.json.contains("max_logstore_id")) ? static_cast<logstore_id_t > (std::stoul(request.json["max_logstore_id"].get<std::string>())): UINT32_MAX;
-    m_id_logstore_map.withRLock([&](auto& id_logstore_map) {
-        for (const auto& [id, lstore] : id_logstore_map) {
-            if(id >= lower_logstore_id && id <= higher_logstore_id)
-            response.json["logstore_id_" + std::to_string(id)] = lstore.m_log_store->get_status(request).json;
-        }
-    });
+    // logstore_id_t lower_logstore_id = (request.json.contains("min_logstore_id"))
+    //     ? static_cast< logstore_id_t >(std::stoul(request.json["min_logstore_id"].get< std::string >()))
+    //     : 0;
+    // logstore_id_t higher_logstore_id = (request.json.contains("max_logstore_id"))
+    //     ? static_cast< logstore_id_t >(std::stoul(request.json["max_logstore_id"].get< std::string >()))
+    //     : UINT32_MAX;
+    // m_id_logstore_map.withRLock([&](auto& id_logstore_map) {
+    //     for (const auto& [id, lstore] : id_logstore_map) {
+    //         if (id >= lower_logstore_id && id <= higher_logstore_id)
+    //             response.json["logstore_id_" + std::to_string(id)] = lstore.m_log_store->get_status(request).json;
```

Comment:
Why does these lines need to be commented?

Reply:
As discussed, we are removing the API for logstore adn use get status instead. Discussed for all review comments.